lib/ansible/modules/find.py

```diff
@@ -201,6 +225,26 @@
       - '^_[0-9]{2,4}_.*.log$'
       - '^[a-z]{1,5}_.*log$'
 
+- name: find only user executable elements in a certain dir
+  find:
+    paths: /opt/myapp
+    permissions:
+      - u+x
+
+- name: find all executable elements in a certain dir
+  find:
+    paths: /opt/myapp
+    permissions:
+      - u+x
+      - g+x
+      - o+x
+
+- name: find all elements owned by user and group example
+  find:
+    paths: /etc
+    user: root
+    group: shadow
+
```

Comment:
a list example for user/group would be nice

Reply:
Good idea! Updated it.